src/assets/css/components/navbar.css

```diff
@@ -0,0 +1,21 @@
+.navbar {
+  height: 100%;
+  display: flex;
+  align-items: center;
+  padding: 0 8px;
+}
+
+.navbar .navbar__list {
+  height: 100%;
+  display: flex;
```

Comment:
I think we can create a display flex class in order to keep it DRY

Reply:
Yes, that is true, but if you take that approach you will need to use it for everything, like colors, font sizes, etc.